### PR TITLE
motdgen: use name and color from os-release

### PR DESCRIPTION
**Pull request by crawford**
image

### CODE REVIEW

**Comment by euank** [comment]
LGTM

**Comment by marineam** [comment]
Idea: How about use $PRETTY_NAME from os-release?

**Comment by marineam** [comment]
Also $ANSI_COLOR

**Comment by crawford** [comment]
Oh shoot. That's a great idea.

**Comment by barakmich** [comment]
Couldn't it also be [blue]Container[/blue] [magenta]Linux[/magenta]?, following the left to right in the new logo?

/literalbikeshed

**Comment by crawford** [comment]
@barakmich no. The colors were chosen by the UX team.

**Comment by crawford** [comment]
@marineam I met you halfway. The pretty name contains the codename (which is almost worthless) and just cluttered up the message. If we did actually give every release a new name, I think it'd be fine to include it.

image

**Comment by philips** [comment]
@crawford probably should use "Container Linux by CoreOS" for the time being. cc @meghanschofield @robszumski

**Comment by crawford** [comment]
Yeah, I was wondering that myself. "CoreOS Container Linux" or just "Container Linux".

**Comment by marineam** [comment]
yeah, $NAME is fine. "By CoreOS" sounds a bit obnoxious. We could do "CoreOS Container Linux" if keeping CoreOS is important but in that case $NAME in os-release should include it. Either way this PR is correct.

File: scripts/motdgen [hunk 1]
```diff
@@ -11,7 +11,7 @@
 source /usr/lib/os-release
 
 mkdir -p /run/coreos
-echo -e "Core\033[38;5;206mO\033[38;5;45mS\033[39m ${GROUP} (${VERSION})" > /run/coreos/motd
+echo -e "\e[${ANSI_COLOR}m${NAME}\e[39m ${GROUP^} (${VERSION})" > /run/coreos/motd
```
Comment (marineam) [inline scripts/motdgen]: Group might be a UUID or other identifier in custom CoreUpdate setups. Just leave the case alone.

**Comment by meghanschofield** [comment]
According to Jim's naming convention write-up, we do want to use "by CoreOS" when it makes sense. But we don't want to be obnoxious. 

For now, I wonder if we want to add the "By CoreOS" since it's a recent change. Sorry @crawford I should have caught that earlier when we were talking colors.

**Comment by meghanschofield** [comment]
In this case, for now, it should be: "Container Linux by CoreOS" 

I'm channeling Jim, and also had some IRL conversations with Brandon and Crawford. This is how we do this thing now, and have been doing it. It's "PRODUCT NAME by CoreOS"

Thanks!

**Pull request merged**